Articles/Setup/Install/install-umbraco-with-vs-code.md

```diff
@@ -0,0 +1,48 @@
+---
+v8-equivalent: "https://github.com/umbraco/UmbracoDocs/blob/main/Getting-Started/Setup/Install/install-umbraco-with-vs-code.md"
+versionFrom: 9.0.0
+verified-against: alpha-4
+state: partial
+updated-links: true
+---
+
+# Install Umbraco with Visual Studio Code
+
+Follow these steps to be up and running with VS Code quickly. The benefit of using VS Code is that it is super quick to get up and running.
```

Comment:
Hm, this might need a bit of rephrasing, we're essentially saying that "getting up and running with VS Code is quick" twice

Reply:
I see what you mean, I just ported this from the [previous article](https://github.com/umbraco/UmbracoDocs/blob/main/Getting-Started/Setup/Install/install-umbraco-with-vs-code.md#install-umbraco-with-visual-studio-code) but I will rephrase it since it sticks out.